lineman/app/components/group_page/group_help_card/group_help_card.haml

```diff
@@ -0,0 +1,17 @@
+%section.group-help-card
```

Comment:
No I think we just leave it for everyone. We want to see traffic from other installs hitting our docs.